torch/_inductor/lowering.py

```diff
@@ -301,6 +301,28 @@ def wrapped(*args, **kwargs):
         return out
 
     aten_fn = get_overloads(aten_fn)
+    core_aten_ops = core_aten_decompositions()
+
+    for fn in aten_fn:
+        if isinstance(
+            fn, (torch._ops.HigherOrderOperator, torch._ops.OpOverloadPacket)
```

Comment:
What `OpOverloadPacket`s and what `callable`s do we hit here?

Reply:
Thanks for asking, actually, `callable(fn)` is wrong here as OpOverload is also a Callable...

Reply:
I'll rewrite everything like that finally:
```python
    for fn in aten_fn:
        if not isinstance(fn, torch._ops.OpOverload):
            continue

        if torch._C.DispatchKey.Autograd in fn.py_kernels:
            if len(get_decompositions([fn])) < 1:
                raise RuntimeError(
                    "Function with Autograd dispatch key in python kernels should have a decomposition"
                )
            if fn not in core_aten_ops.keys():
                raise RuntimeError(
                    f"Trying to register a lowering for {str(fn)}, which has a "
                    "py_impl Autograd key. A function with py_impl Autograd dispatch "
                    "key should be in core_aten_ops and should not have a lowering."
                )
```
Any comments, suggestions ?

Reply:
What fns are not OpOverloads? We have High Order Ops tha we shoudl skip, but what other objects do we get? We also allow for registering ops for `OpOverloadPacket`s and these are very commonly used, so we should also check those at least.

Nit `if not get_decompositions([fn]):`...

Reply:
Here is a set of all types except OpOverload passing into register_lowering:
```
<class 'torch.tensortype'>
<class 'torch._higher_order_ops.triton_kernel_wrap.TritonKernelWrapperFunctional'>
<class 'torch._higher_order_ops.auto_functionalize.AutoFunctionalized'>
<class 'torch._higher_order_ops.triton_kernel_wrap.TritonKernelWrapperMutation'>
<class 'torch._ops.OpOverloadPacket'>
<class 'builtin_function_or_method'>
<class 'function'>
<class 'torch._ops.HigherOrderOperator'>
```

<details>
<summary>
Here is a complete list for today of non-OpOverload registered function
</summary>

```
# print(fn, type(fn), aten_fn)
prims.convert_element_type <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.convert_element_type')>, <OpOverload(op='prims.convert_element_type', overload='default')>]
prims.device_put <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.device_put')>, <OpOverload(op='prims.device_put', overload='default')>]
aten.frexp <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.frexp')>, <OpOverload(op='aten.frexp', overload='Tensor')>, <OpOverload(op='aten.frexp', overload='Tensor_out')>, <OpOverload(op='aten.frexp', overload='default')>]
prims.frexp <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.frexp')>, <OpOverload(op='prims.frexp', overload='default')>]
aten.where <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.where')>, <OpOverload(op='aten.where', overload='self')>, <OpOverload(op='aten.where', overload='ScalarOther')>, <OpOverload(op='aten.where', overload='ScalarSelf')>, <OpOverload(op='aten.where', overload='Scalar')>, <OpOverload(op='aten.where', overload='default')>, <OpOverload(op='aten.where', overload='self_out')>]
aten.broadcast_tensors <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.broadcast_tensors')>, <OpOverload(op='aten.broadcast_tensors', overload='default')>]
aten.alias <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.alias')>, <OpOverloadPacket(op='aten.detach')>, <OpOverloadPacket(op='aten.detach_')>, <OpOverloadPacket(op='aten.lift')>, <OpOverloadPacket(op='prims.view_of')>, <OpOverload(op='aten.alias', overload='default')>, <OpOverload(op='aten.detach', overload='default')>, <OpOverload(op='aten.detach_', overload='default')>, <OpOverload(op='aten.lift', overload='default')>, <OpOverload(op='aten.lift', overload='out')>, <OpOverload(op='prims.view_of', overload='default')>]
aten.detach <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.alias')>, <OpOverloadPacket(op='aten.detach')>, <OpOverloadPacket(op='aten.detach_')>, <OpOverloadPacket(op='aten.lift')>, <OpOverloadPacket(op='prims.view_of')>, <OpOverload(op='aten.alias', overload='default')>, <OpOverload(op='aten.detach', overload='default')>, <OpOverload(op='aten.detach_', overload='default')>, <OpOverload(op='aten.lift', overload='default')>, <OpOverload(op='aten.lift', overload='out')>, <OpOverload(op='prims.view_of', overload='default')>]
aten.detach_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.alias')>, <OpOverloadPacket(op='aten.detach')>, <OpOverloadPacket(op='aten.detach_')>, <OpOverloadPacket(op='aten.lift')>, <OpOverloadPacket(op='prims.view_of')>, <OpOverload(op='aten.alias', overload='default')>, <OpOverload(op='aten.detach', overload='default')>, <OpOverload(op='aten.detach_', overload='default')>, <OpOverload(op='aten.lift', overload='default')>, <OpOverload(op='aten.lift', overload='out')>, <OpOverload(op='prims.view_of', overload='default')>]
aten.lift <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.alias')>, <OpOverloadPacket(op='aten.detach')>, <OpOverloadPacket(op='aten.detach_')>, <OpOverloadPacket(op='aten.lift')>, <OpOverloadPacket(op='prims.view_of')>, <OpOverload(op='aten.alias', overload='default')>, <OpOverload(op='aten.detach', overload='default')>, <OpOverload(op='aten.detach_', overload='default')>, <OpOverload(op='aten.lift', overload='default')>, <OpOverload(op='aten.lift', overload='out')>, <OpOverload(op='prims.view_of', overload='default')>]
prims.view_of <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.alias')>, <OpOverloadPacket(op='aten.detach')>, <OpOverloadPacket(op='aten.detach_')>, <OpOverloadPacket(op='aten.lift')>, <OpOverloadPacket(op='prims.view_of')>, <OpOverload(op='aten.alias', overload='default')>, <OpOverload(op='aten.detach', overload='default')>, <OpOverload(op='aten.detach_', overload='default')>, <OpOverload(op='aten.lift', overload='default')>, <OpOverload(op='aten.lift', overload='out')>, <OpOverload(op='prims.view_of', overload='default')>]
aten.lift_fresh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.lift_fresh')>, <OpOverload(op='aten.lift_fresh', overload='default')>]
aten.squeeze <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.squeeze')>, <OpOverload(op='aten.squeeze', overload='default')>, <OpOverload(op='aten.squeeze', overload='dim')>, <OpOverload(op='aten.squeeze', overload='dims')>, <OpOverload(op='aten.squeeze', overload='dimname')>]
aten.squeeze_copy <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.squeeze_copy')>, <OpOverload(op='aten.squeeze_copy', overload='out')>, <OpOverload(op='aten.squeeze_copy', overload='dim_out')>, <OpOverload(op='aten.squeeze_copy', overload='dims_out')>, <OpOverload(op='aten.squeeze_copy', overload='default')>, <OpOverload(op='aten.squeeze_copy', overload='dim')>, <OpOverload(op='aten.squeeze_copy', overload='dims')>]
aten.squeeze_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.squeeze_')>, <OpOverload(op='aten.squeeze_', overload='default')>, <OpOverload(op='aten.squeeze_', overload='dim')>, <OpOverload(op='aten.squeeze_', overload='dims')>, <OpOverload(op='aten.squeeze_', overload='dimname')>]
aten.isinf <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.isinf')>, <OpOverload(op='aten.isinf', overload='default')>, <OpOverload(op='aten.isinf', overload='out')>, <OpOverload(op='aten.isinf', overload='float')>, <OpOverload(op='aten.isinf', overload='complex')>]
aten.isnan <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.isnan')>, <OpOverload(op='aten.isnan', overload='default')>, <OpOverload(op='aten.isnan', overload='out')>, <OpOverload(op='aten.isnan', overload='float')>, <OpOverload(op='aten.isnan', overload='complex')>]
aten.ceil <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.ceil')>, <OpOverload(op='aten.ceil', overload='default')>, <OpOverload(op='aten.ceil', overload='out')>, <OpOverload(op='aten.ceil', overload='int')>, <OpOverload(op='aten.ceil', overload='float')>, <OpOverload(op='aten.ceil', overload='Scalar')>]
aten.floor <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.floor')>, <OpOverload(op='aten.floor', overload='default')>, <OpOverload(op='aten.floor', overload='out')>, <OpOverload(op='aten.floor', overload='int')>, <OpOverload(op='aten.floor', overload='float')>, <OpOverload(op='aten.floor', overload='Scalar')>]
aten.trunc <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.trunc')>, <OpOverload(op='aten.trunc', overload='default')>, <OpOverload(op='aten.trunc', overload='out')>]
aten.expand <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.expand')>, <OpOverload(op='aten.expand', overload='default')>]
prims.broadcast_in_dim <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.broadcast_in_dim')>, <OpOverload(op='prims.broadcast_in_dim', overload='default')>]
aten.expand_as <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.expand_as')>, <OpOverload(op='aten.expand_as', overload='default')>]
aten.repeat <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.repeat')>, <OpOverload(op='aten.repeat', overload='default')>, <OpOverload(op='aten.repeat', overload='out')>]
aten.reshape <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.reshape')>, <OpOverload(op='aten.reshape', overload='default')>]
aten.view <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.view')>, <OpOverload(op='aten.view', overload='default')>]
aten._unsafe_view <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._unsafe_view')>, <OpOverload(op='aten._unsafe_view', overload='default')>, <OpOverload(op='aten._unsafe_view', overload='out')>]
aten.permute <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.permute')>, <OpOverload(op='aten.permute', overload='default')>]
aten.slice <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.slice')>, <OpOverload(op='aten.slice', overload='Tensor')>, <OpOverload(op='aten.slice', overload='t')>, <OpOverload(op='aten.slice', overload='str')>]
aten.as_strided <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.as_strided')>, <OpOverload(op='aten.as_strided', overload='default')>]
aten.as_strided_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.as_strided_')>, <OpOverload(op='aten.as_strided_', overload='default')>]
aten.as_strided_copy <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.as_strided_copy')>, <OpOverload(op='aten.as_strided_copy', overload='out')>, <OpOverload(op='aten.as_strided_copy', overload='default')>]
quantized_decomposed.quantize_per_channel <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='quantized_decomposed.quantize_per_channel')>, <OpOverload(op='quantized_decomposed.quantize_per_channel', overload='default')>]
quantized_decomposed.dequantize_per_channel <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='quantized_decomposed.dequantize_per_channel')>, <OpOverload(op='quantized_decomposed.dequantize_per_channel', overload='default')>]
aten.cat <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.cat')>, <OpOverload(op='aten.cat', overload='default')>, <OpOverload(op='aten.cat', overload='names')>, <OpOverload(op='aten.cat', overload='names_out')>, <OpOverload(op='aten.cat', overload='out')>]
aten.diagonal <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.diagonal')>, <OpOverload(op='aten.diagonal', overload='default')>, <OpOverload(op='aten.diagonal', overload='Dimname')>]
aten.diagonal_copy <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.diagonal_copy')>, <OpOverload(op='aten.diagonal_copy', overload='default')>, <OpOverload(op='aten.diagonal_copy', overload='out')>]
aten.diagonal_scatter <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.diagonal_scatter')>, <OpOverload(op='aten.diagonal_scatter', overload='default')>, <OpOverload(op='aten.diagonal_scatter', overload='out')>]
aten.select <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.select')>, <OpOverload(op='aten.select', overload='Dimname')>, <OpOverload(op='aten.select', overload='int')>, <OpOverload(op='aten.select', overload='t')>]
aten.split <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.split')>, <OpOverload(op='aten.split', overload='Tensor')>, <OpOverload(op='aten.split', overload='sizes')>, <OpOverload(op='aten.split', overload='str')>, <OpOverload(op='aten.split', overload='default')>]
aten.split_with_sizes <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.split_with_sizes')>, <OpOverload(op='aten.split_with_sizes', overload='default')>]
aten.unbind <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.unbind')>, <OpOverload(op='aten.unbind', overload='int')>, <OpOverload(op='aten.unbind', overload='Dimname')>]
aten.unfold <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.unfold')>, <OpOverload(op='aten.unfold', overload='default')>]
aten.unsqueeze <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.unsqueeze')>, <OpOverload(op='aten.unsqueeze', overload='default')>]
aten.unsqueeze_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.unsqueeze_')>, <OpOverload(op='aten.unsqueeze_', overload='default')>]
aten.glu <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.glu')>, <OpOverload(op='aten.glu', overload='default')>, <OpOverload(op='aten.glu', overload='out')>]
rngprims.philox_rand <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='rngprims.philox_rand')>, <OpOverload(op='rngprims.philox_rand', overload='default')>]
aten.native_dropout <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.native_dropout')>, <OpOverload(op='aten.native_dropout', overload='default')>, <OpOverload(op='aten.native_dropout', overload='out')>]
aten.bernoulli_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bernoulli_')>, <OpOverload(op='aten.bernoulli_', overload='Tensor')>, <OpOverload(op='aten.bernoulli_', overload='float')>]
aten._foobar <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._foobar')>, <OpOverload(op='aten._foobar', overload='default')>, <OpOverload(op='aten._foobar', overload='out')>]
aten.rand <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.rand')>, <OpOverload(op='aten.rand', overload='default')>, <OpOverload(op='aten.rand', overload='generator')>, <OpOverload(op='aten.rand', overload='names')>, <OpOverload(op='aten.rand', overload='generator_with_names')>, <OpOverload(op='aten.rand', overload='out')>, <OpOverload(op='aten.rand', overload='generator_out')>, <OpOverload(op='aten.rand', overload='names_out')>, <OpOverload(op='aten.rand', overload='generator_with_names_out')>]
aten.randn <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.randn')>, <OpOverload(op='aten.randn', overload='default')>, <OpOverload(op='aten.randn', overload='generator')>, <OpOverload(op='aten.randn', overload='names')>, <OpOverload(op='aten.randn', overload='generator_with_names')>, <OpOverload(op='aten.randn', overload='out')>, <OpOverload(op='aten.randn', overload='generator_out')>, <OpOverload(op='aten.randn', overload='names_out')>, <OpOverload(op='aten.randn', overload='generator_with_names_out')>]
aten.bucketize <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bucketize')>, <OpOverload(op='aten.bucketize', overload='Tensor')>, <OpOverload(op='aten.bucketize', overload='Scalar')>, <OpOverload(op='aten.bucketize', overload='Tensor_out')>, <OpOverload(op='aten.bucketize', overload='Scalar_out')>]
run_and_save_rng_state <class 'torch._ops.HigherOrderOperator'> [<torch._ops.HigherOrderOperator object at 0x7ff1ff683400>]
run_with_rng_state <class 'torch._ops.HigherOrderOperator'> [<torch._ops.HigherOrderOperator object at 0x7ff1ff685040>]
aten.copy <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.copy')>, <OpOverload(op='aten.copy', overload='out')>, <OpOverload(op='aten.copy', overload='default')>, <OpOverload(op='aten.copy', overload='t')>, <OpOverload(op='aten.copy', overload='Dict_str')>, <OpOverload(op='aten.copy', overload='Dict_int')>, <OpOverload(op='aten.copy', overload='Dict_bool')>, <OpOverload(op='aten.copy', overload='Dict_float')>, <OpOverload(op='aten.copy', overload='Dict_complex')>, <OpOverload(op='aten.copy', overload='Dict_Tensor')>]
aten.clone <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.clone')>, <OpOverload(op='aten.clone', overload='default')>, <OpOverload(op='aten.clone', overload='out')>]
aten.lift_fresh_copy <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.lift_fresh_copy')>, <OpOverload(op='aten.lift_fresh_copy', overload='default')>, <OpOverload(op='aten.lift_fresh_copy', overload='out')>]
prims.iota <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.iota')>, <OpOverload(op='prims.iota', overload='default')>]
aten.select_scatter <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.select_scatter')>, <OpOverload(op='aten.select_scatter', overload='default')>, <OpOverload(op='aten.select_scatter', overload='out')>]
aten.slice_scatter <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.slice_scatter')>, <OpOverload(op='aten.slice_scatter', overload='default')>, <OpOverload(op='aten.slice_scatter', overload='out')>]
<built-in method tensor of type object at 0x7ff28161cdc0> <class 'builtin_function_or_method'> [<built-in method tensor of type object at 0x7ff28161cdc0>, <OpOverloadPacket(op='aten.scalar_tensor')>, <OpOverload(op='aten.scalar_tensor', overload='default')>, <OpOverload(op='aten.scalar_tensor', overload='out')>]
aten.scalar_tensor <class 'torch._ops.OpOverloadPacket'> [<built-in method tensor of type object at 0x7ff28161cdc0>, <OpOverloadPacket(op='aten.scalar_tensor')>, <OpOverload(op='aten.scalar_tensor', overload='default')>, <OpOverload(op='aten.scalar_tensor', overload='out')>]
<built-in method as_tensor of type object at 0x7ff28161cdc0> <class 'builtin_function_or_method'> [<built-in method as_tensor of type object at 0x7ff28161cdc0>]
<class 'torch.LongTensor'> <class 'torch.tensortype'> [<class 'torch.LongTensor'>]
aten._local_scalar_dense <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._local_scalar_dense')>, <OpOverload(op='aten._local_scalar_dense', overload='default')>]
aten._assert_scalar <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._assert_scalar')>, <OpOverload(op='aten._assert_scalar', overload='default')>]
aten.full_like <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.full_like')>, <OpOverload(op='aten.full_like', overload='default')>, <OpOverload(op='aten.full_like', overload='out')>]
<built-in method empty of type object at 0x7ff28161cdc0> <class 'builtin_function_or_method'> [<built-in method empty of type object at 0x7ff28161cdc0>, <OpOverloadPacket(op='aten.empty')>, <OpOverload(op='aten.empty', overload='memory_format')>, <OpOverload(op='aten.empty', overload='out')>, <OpOverload(op='aten.empty', overload='names')>, <OpOverload(op='aten.empty', overload='names_out')>]
aten.empty <class 'torch._ops.OpOverloadPacket'> [<built-in method empty of type object at 0x7ff28161cdc0>, <OpOverloadPacket(op='aten.empty')>, <OpOverload(op='aten.empty', overload='memory_format')>, <OpOverload(op='aten.empty', overload='out')>, <OpOverload(op='aten.empty', overload='names')>, <OpOverload(op='aten.empty', overload='names_out')>]
aten.empty_like <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.empty_like')>, <OpOverload(op='aten.empty_like', overload='default')>, <OpOverload(op='aten.empty_like', overload='out')>]
aten.new_empty <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.new_empty')>, <OpOverload(op='aten.new_empty', overload='default')>, <OpOverload(op='aten.new_empty', overload='out')>]
aten.empty_strided <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.empty_strided')>, <OpOverload(op='aten.empty_strided', overload='default')>, <OpOverload(op='aten.empty_strided', overload='out')>]
aten.new_empty_strided <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.new_empty_strided')>, <OpOverload(op='aten.new_empty_strided', overload='default')>, <OpOverload(op='aten.new_empty_strided', overload='out')>]
<built-in method full of type object at 0x7ff28161cdc0> <class 'builtin_function_or_method'> [<built-in method full of type object at 0x7ff28161cdc0>, <OpOverloadPacket(op='aten.full')>, <OpOverload(op='aten.full', overload='names')>, <OpOverload(op='aten.full', overload='default')>, <OpOverload(op='aten.full', overload='names_out')>, <OpOverload(op='aten.full', overload='out')>]
aten.full <class 'torch._ops.OpOverloadPacket'> [<built-in method full of type object at 0x7ff28161cdc0>, <OpOverloadPacket(op='aten.full')>, <OpOverload(op='aten.full', overload='names')>, <OpOverload(op='aten.full', overload='default')>, <OpOverload(op='aten.full', overload='names_out')>, <OpOverload(op='aten.full', overload='out')>]
aten.gather <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.gather')>, <OpOverload(op='aten.gather', overload='default')>, <OpOverload(op='aten.gather', overload='out')>, <OpOverload(op='aten.gather', overload='dimname')>, <OpOverload(op='aten.gather', overload='dimname_out')>]
aten.embedding <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.embedding')>, <OpOverload(op='aten.embedding', overload='default')>, <OpOverload(op='aten.embedding', overload='out')>]
aten.index <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.index')>, <OpOverload(op='aten.index', overload='Tensor')>, <OpOverload(op='aten.index', overload='Tensor_out')>, <OpOverload(op='aten.index', overload='Tensor_hacked_twin')>, <OpOverload(op='aten.index', overload='str')>, <OpOverload(op='aten.index', overload='list_int')>, <OpOverload(op='aten.index', overload='list_float')>, <OpOverload(op='aten.index', overload='list_bool')>, <OpOverload(op='aten.index', overload='list_Tensor')>, <OpOverload(op='aten.index', overload='list_str')>]
aten._unsafe_index <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._unsafe_index')>, <OpOverload(op='aten._unsafe_index', overload='Tensor')>, <OpOverload(op='aten._unsafe_index', overload='Tensor_hacked_twin')>]
aten.index_put <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.index_put')>, <OpOverload(op='aten.index_put', overload='default')>, <OpOverload(op='aten.index_put', overload='out')>, <OpOverload(op='aten.index_put', overload='hacked_twin')>]
aten._unsafe_index_put <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._unsafe_index_put')>, <OpOverload(op='aten._unsafe_index_put', overload='default')>, <OpOverload(op='aten._unsafe_index_put', overload='hacked_twin')>]
aten.index_put_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.index_put_')>, <OpOverload(op='aten.index_put_', overload='default')>, <OpOverload(op='aten.index_put_', overload='hacked_twin')>]
aten.as_strided_scatter <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.as_strided_scatter')>, <OpOverload(op='aten.as_strided_scatter', overload='default')>, <OpOverload(op='aten.as_strided_scatter', overload='out')>]
aten.scatter <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.scatter')>, <OpOverload(op='aten.scatter', overload='value')>, <OpOverload(op='aten.scatter', overload='src')>, <OpOverload(op='aten.scatter', overload='reduce')>, <OpOverload(op='aten.scatter', overload='value_reduce')>, <OpOverload(op='aten.scatter', overload='src_out')>, <OpOverload(op='aten.scatter', overload='value_out')>, <OpOverload(op='aten.scatter', overload='reduce_out')>, <OpOverload(op='aten.scatter', overload='value_reduce_out')>, <OpOverload(op='aten.scatter', overload='dimname_src')>, <OpOverload(op='aten.scatter', overload='dimname_value')>]
aten.scatter_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.scatter_')>, <OpOverload(op='aten.scatter_', overload='src')>, <OpOverload(op='aten.scatter_', overload='value')>, <OpOverload(op='aten.scatter_', overload='reduce')>, <OpOverload(op='aten.scatter_', overload='value_reduce')>]
aten.scatter_add <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.scatter_add')>, <OpOverload(op='aten.scatter_add', overload='default')>, <OpOverload(op='aten.scatter_add', overload='out')>, <OpOverload(op='aten.scatter_add', overload='dimname')>]
aten.scatter_add_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.scatter_add_')>, <OpOverload(op='aten.scatter_add_', overload='default')>]
aten.scatter_reduce <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.scatter_reduce')>, <OpOverload(op='aten.scatter_reduce', overload='two')>, <OpOverload(op='aten.scatter_reduce', overload='two_out')>]
aten.scatter_reduce_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.scatter_reduce_')>, <OpOverload(op='aten.scatter_reduce_', overload='two')>]
aten.reflection_pad3d_backward <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.reflection_pad3d_backward')>, <OpOverload(op='aten.reflection_pad3d_backward', overload='default')>, <OpOverload(op='aten.reflection_pad3d_backward', overload='grad_input')>]
aten.reflection_pad2d_backward <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.reflection_pad2d_backward')>, <OpOverload(op='aten.reflection_pad2d_backward', overload='default')>, <OpOverload(op='aten.reflection_pad2d_backward', overload='grad_input')>]
aten.reflection_pad1d_backward <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.reflection_pad1d_backward')>, <OpOverload(op='aten.reflection_pad1d_backward', overload='default')>, <OpOverload(op='aten.reflection_pad1d_backward', overload='grad_input')>]
aten.constant_pad_nd <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.constant_pad_nd')>, <OpOverload(op='aten.constant_pad_nd', overload='default')>, <OpOverload(op='aten.constant_pad_nd', overload='out')>]
aten.max_pool2d_with_indices <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.max_pool2d_with_indices')>, <OpOverload(op='aten.max_pool2d_with_indices', overload='default')>, <OpOverload(op='aten.max_pool2d_with_indices', overload='out')>]
aten.max_pool2d_with_indices_backward <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.max_pool2d_with_indices_backward')>, <OpOverload(op='aten.max_pool2d_with_indices_backward', overload='default')>, <OpOverload(op='aten.max_pool2d_with_indices_backward', overload='grad_input')>]
aten._adaptive_avg_pool2d <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._adaptive_avg_pool2d')>, <OpOverload(op='aten._adaptive_avg_pool2d', overload='default')>, <OpOverload(op='aten._adaptive_avg_pool2d', overload='out')>]
aten.adaptive_max_pool2d <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.adaptive_max_pool2d')>, <OpOverload(op='aten.adaptive_max_pool2d', overload='default')>, <OpOverload(op='aten.adaptive_max_pool2d', overload='out')>]
aten.fractional_max_pool2d <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.fractional_max_pool2d')>, <OpOverload(op='aten.fractional_max_pool2d', overload='default')>, <OpOverload(op='aten.fractional_max_pool2d', overload='output')>]
aten.avg_pool2d <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.avg_pool2d')>, <OpOverload(op='aten.avg_pool2d', overload='default')>, <OpOverload(op='aten.avg_pool2d', overload='out')>]
aten.avg_pool3d <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.avg_pool3d')>, <OpOverload(op='aten.avg_pool3d', overload='default')>, <OpOverload(op='aten.avg_pool3d', overload='out')>]
aten.avg_pool2d_backward <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.avg_pool2d_backward')>, <OpOverload(op='aten.avg_pool2d_backward', overload='default')>, <OpOverload(op='aten.avg_pool2d_backward', overload='grad_input')>]
aten.mean <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.mean')>, <OpOverload(op='aten.mean', overload='default')>, <OpOverload(op='aten.mean', overload='dim')>, <OpOverload(op='aten.mean', overload='names_dim')>, <OpOverload(op='aten.mean', overload='names_out')>, <OpOverload(op='aten.mean', overload='out')>]
aten.var <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.var')>, <OpOverloadPacket(op='prims.var')>, <OpOverload(op='aten.var', overload='default')>, <OpOverload(op='aten.var', overload='dim')>, <OpOverload(op='aten.var', overload='correction')>, <OpOverload(op='aten.var', overload='names_dim')>, <OpOverload(op='aten.var', overload='names_out')>, <OpOverload(op='aten.var', overload='out')>, <OpOverload(op='aten.var', overload='correction_out')>, <OpOverload(op='aten.var', overload='correction_names')>, <OpOverload(op='aten.var', overload='correction_names_out')>, <OpOverload(op='prims.var', overload='default')>]
prims.var <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.var')>, <OpOverloadPacket(op='prims.var')>, <OpOverload(op='aten.var', overload='default')>, <OpOverload(op='aten.var', overload='dim')>, <OpOverload(op='aten.var', overload='correction')>, <OpOverload(op='aten.var', overload='names_dim')>, <OpOverload(op='aten.var', overload='names_out')>, <OpOverload(op='aten.var', overload='out')>, <OpOverload(op='aten.var', overload='correction_out')>, <OpOverload(op='aten.var', overload='correction_names')>, <OpOverload(op='aten.var', overload='correction_names_out')>, <OpOverload(op='prims.var', overload='default')>]
aten.var_mean <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.var_mean')>, <OpOverload(op='aten.var_mean', overload='default')>, <OpOverload(op='aten.var_mean', overload='dim')>, <OpOverload(op='aten.var_mean', overload='correction')>, <OpOverload(op='aten.var_mean', overload='names_dim')>, <OpOverload(op='aten.var_mean', overload='correction_names')>, <OpOverload(op='aten.var_mean', overload='correction_out')>]
aten.pow <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.pow')>, <OpOverload(op='aten.pow', overload='Tensor_Tensor')>, <OpOverload(op='aten.pow', overload='Tensor_Scalar')>, <OpOverload(op='aten.pow', overload='Scalar')>, <OpOverload(op='aten.pow', overload='Scalar_out')>, <OpOverload(op='aten.pow', overload='Tensor_Scalar_out')>, <OpOverload(op='aten.pow', overload='Tensor_Tensor_out')>, <OpOverload(op='aten.pow', overload='int')>, <OpOverload(op='aten.pow', overload='complex')>, <OpOverload(op='aten.pow', overload='float')>, <OpOverload(op='aten.pow', overload='int_float')>, <OpOverload(op='aten.pow', overload='float_int')>, <OpOverload(op='aten.pow', overload='float_complex')>, <OpOverload(op='aten.pow', overload='complex_float')>, <OpOverload(op='aten.pow', overload='Scalar_Scalar')>, <OpOverload(op='aten.pow', overload='int_to_int')>]
aten.fill_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.fill_')>, <OpOverload(op='aten.fill_', overload='Scalar')>, <OpOverload(op='aten.fill_', overload='Tensor')>]
aten.copy_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.copy_')>, <OpOverload(op='aten.copy_', overload='default')>, <OpOverload(op='aten.copy_', overload='Tensor')>, <OpOverload(op='aten.copy_', overload='int')>, <OpOverload(op='aten.copy_', overload='float')>]
aten.div <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.div')>, <OpOverload(op='aten.div', overload='Tensor')>, <OpOverload(op='aten.div', overload='Scalar')>, <OpOverload(op='aten.div', overload='Tensor_mode')>, <OpOverload(op='aten.div', overload='Scalar_mode')>, <OpOverload(op='aten.div', overload='out')>, <OpOverload(op='aten.div', overload='out_mode')>, <OpOverload(op='aten.div', overload='Scalar_out')>, <OpOverload(op='aten.div', overload='Scalar_mode_out')>, <OpOverload(op='aten.div', overload='int')>, <OpOverload(op='aten.div', overload='complex')>, <OpOverload(op='aten.div', overload='float')>, <OpOverload(op='aten.div', overload='default')>]
aten.mul <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.mul')>, <OpOverload(op='aten.mul', overload='Tensor')>, <OpOverload(op='aten.mul', overload='Scalar')>, <OpOverload(op='aten.mul', overload='out')>, <OpOverload(op='aten.mul', overload='Scalar_out')>, <OpOverload(op='aten.mul', overload='left_t')>, <OpOverload(op='aten.mul', overload='right_')>, <OpOverload(op='aten.mul', overload='int')>, <OpOverload(op='aten.mul', overload='complex')>, <OpOverload(op='aten.mul', overload='float')>, <OpOverload(op='aten.mul', overload='int_complex')>, <OpOverload(op='aten.mul', overload='complex_int')>, <OpOverload(op='aten.mul', overload='float_complex')>, <OpOverload(op='aten.mul', overload='complex_float')>, <OpOverload(op='aten.mul', overload='int_float')>, <OpOverload(op='aten.mul', overload='float_int')>, <OpOverload(op='aten.mul', overload='default')>]
prims.div <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.div')>, <OpOverload(op='prims.div', overload='default')>]
aten.true_divide <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.true_divide')>, <OpOverload(op='aten.div', overload='Tensor')>, <OpOverload(op='aten.true_divide', overload='Tensor')>, <OpOverload(op='aten.true_divide', overload='Scalar')>, <OpOverload(op='aten.true_divide', overload='out')>]
aten.fmod <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.fmod')>, <OpOverloadPacket(op='prims.fmod')>, <OpOverload(op='aten.fmod', overload='Tensor')>, <OpOverload(op='aten.fmod', overload='Scalar')>, <OpOverload(op='aten.fmod', overload='Tensor_out')>, <OpOverload(op='aten.fmod', overload='Scalar_out')>, <OpOverload(op='aten.fmod', overload='int')>, <OpOverload(op='aten.fmod', overload='float')>, <OpOverload(op='aten.fmod', overload='int_float')>, <OpOverload(op='aten.fmod', overload='float_int')>, <OpOverload(op='aten.fmod', overload='default')>, <OpOverload(op='prims.fmod', overload='default')>]
prims.fmod <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.fmod')>, <OpOverloadPacket(op='prims.fmod')>, <OpOverload(op='aten.fmod', overload='Tensor')>, <OpOverload(op='aten.fmod', overload='Scalar')>, <OpOverload(op='aten.fmod', overload='Tensor_out')>, <OpOverload(op='aten.fmod', overload='Scalar_out')>, <OpOverload(op='aten.fmod', overload='int')>, <OpOverload(op='aten.fmod', overload='float')>, <OpOverload(op='aten.fmod', overload='int_float')>, <OpOverload(op='aten.fmod', overload='float_int')>, <OpOverload(op='aten.fmod', overload='default')>, <OpOverload(op='prims.fmod', overload='default')>]
aten.rsqrt <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.rsqrt')>, <OpOverload(op='aten.rsqrt', overload='default')>, <OpOverload(op='aten.rsqrt', overload='out')>]
aten.sum <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sum')>, <OpOverloadPacket(op='prims.sum')>, <OpOverload(op='aten.sum', overload='dim_IntList')>, <OpOverload(op='aten.sum', overload='default')>, <OpOverload(op='aten.sum', overload='dim_DimnameList')>, <OpOverload(op='aten.sum', overload='DimnameList_out')>, <OpOverload(op='aten.sum', overload='IntList_out')>, <OpOverload(op='aten.sum', overload='out')>, <OpOverload(op='aten.sum', overload='int')>, <OpOverload(op='aten.sum', overload='float')>, <OpOverload(op='aten.sum', overload='complex')>, <OpOverload(op='aten.sum', overload='bool')>, <OpOverload(op='prims.sum', overload='default')>]
prims.sum <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sum')>, <OpOverloadPacket(op='prims.sum')>, <OpOverload(op='aten.sum', overload='dim_IntList')>, <OpOverload(op='aten.sum', overload='default')>, <OpOverload(op='aten.sum', overload='dim_DimnameList')>, <OpOverload(op='aten.sum', overload='DimnameList_out')>, <OpOverload(op='aten.sum', overload='IntList_out')>, <OpOverload(op='aten.sum', overload='out')>, <OpOverload(op='aten.sum', overload='int')>, <OpOverload(op='aten.sum', overload='float')>, <OpOverload(op='aten.sum', overload='complex')>, <OpOverload(op='aten.sum', overload='bool')>, <OpOverload(op='prims.sum', overload='default')>]
aten.cumsum <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.cumsum')>, <OpOverload(op='aten.cumsum', overload='default')>, <OpOverload(op='aten.cumsum', overload='dimname')>, <OpOverload(op='aten.cumsum', overload='dimname_out')>, <OpOverload(op='aten.cumsum', overload='out')>]
aten.cumprod <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.cumprod')>, <OpOverload(op='aten.cumprod', overload='default')>, <OpOverload(op='aten.cumprod', overload='dimname')>, <OpOverload(op='aten.cumprod', overload='dimname_out')>, <OpOverload(op='aten.cumprod', overload='out')>]
aten.logcumsumexp <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.logcumsumexp')>, <OpOverload(op='aten.logcumsumexp', overload='default')>, <OpOverload(op='aten.logcumsumexp', overload='dimname')>, <OpOverload(op='aten.logcumsumexp', overload='dimname_out')>, <OpOverload(op='aten.logcumsumexp', overload='out')>]
aten.cummax <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.cummax')>, <OpOverload(op='aten.cummax', overload='default')>, <OpOverload(op='aten.cummax', overload='dimname')>, <OpOverload(op='aten.cummax', overload='dimname_out')>, <OpOverload(op='aten.cummax', overload='out')>]
aten.cummin <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.cummin')>, <OpOverload(op='aten.cummin', overload='default')>, <OpOverload(op='aten.cummin', overload='dimname')>, <OpOverload(op='aten.cummin', overload='dimname_out')>, <OpOverload(op='aten.cummin', overload='out')>]
aten.prod <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.prod')>, <OpOverload(op='aten.prod', overload='default')>, <OpOverload(op='aten.prod', overload='dim_int')>, <OpOverload(op='aten.prod', overload='dim_Dimname')>, <OpOverload(op='aten.prod', overload='Dimname_out')>, <OpOverload(op='aten.prod', overload='int_out')>, <OpOverload(op='aten.prod', overload='out')>]
aten.any <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.any')>, <OpOverload(op='aten.any', overload='default')>, <OpOverload(op='aten.any', overload='dim')>, <OpOverload(op='aten.any', overload='dims')>, <OpOverload(op='aten.any', overload='out')>, <OpOverload(op='aten.any', overload='dims_out')>, <OpOverload(op='aten.any', overload='all_out')>, <OpOverload(op='aten.any', overload='dimname')>, <OpOverload(op='aten.any', overload='dimname_out')>, <OpOverload(op='aten.any', overload='str')>, <OpOverload(op='aten.any', overload='int')>, <OpOverload(op='aten.any', overload='float')>, <OpOverload(op='aten.any', overload='bool')>]
aten.max <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.max')>, <OpOverload(op='aten.max', overload='other')>, <OpOverload(op='aten.max', overload='default')>, <OpOverload(op='aten.max', overload='dim')>, <OpOverload(op='aten.max', overload='dim_max')>, <OpOverload(op='aten.max', overload='names_dim')>, <OpOverload(op='aten.max', overload='names_dim_max')>, <OpOverload(op='aten.max', overload='unary_out')>, <OpOverload(op='aten.max', overload='out')>]
aten.min <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.min')>, <OpOverload(op='aten.min', overload='other')>, <OpOverload(op='aten.min', overload='default')>, <OpOverload(op='aten.min', overload='dim')>, <OpOverload(op='aten.min', overload='dim_min')>, <OpOverload(op='aten.min', overload='names_dim')>, <OpOverload(op='aten.min', overload='names_dim_min')>, <OpOverload(op='aten.min', overload='unary_out')>, <OpOverload(op='aten.min', overload='out')>]
prims.xor_sum <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.xor_sum')>, <OpOverload(op='prims.xor_sum', overload='default')>]
aten.amax <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.amax')>, <OpOverload(op='aten.amax', overload='default')>, <OpOverload(op='aten.amax', overload='out')>]
aten.amin <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.amin')>, <OpOverload(op='aten.amin', overload='default')>, <OpOverload(op='aten.amin', overload='out')>]
aten.argmax <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.argmax')>, <OpOverload(op='aten.argmax', overload='default')>, <OpOverload(op='aten.argmax', overload='out')>]
aten.argmin <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.argmin')>, <OpOverload(op='aten.argmin', overload='default')>, <OpOverload(op='aten.argmin', overload='out')>]
aten.add <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.add')>, <OpOverload(op='aten.add', overload='Tensor')>, <OpOverload(op='aten.add', overload='Scalar')>, <OpOverload(op='aten.add', overload='out')>, <OpOverload(op='aten.add', overload='Scalar_out')>, <OpOverload(op='aten.add', overload='t')>, <OpOverload(op='aten.add', overload='str')>, <OpOverload(op='aten.add', overload='int')>, <OpOverload(op='aten.add', overload='complex')>, <OpOverload(op='aten.add', overload='float')>, <OpOverload(op='aten.add', overload='int_complex')>, <OpOverload(op='aten.add', overload='complex_int')>, <OpOverload(op='aten.add', overload='float_complex')>, <OpOverload(op='aten.add', overload='complex_float')>, <OpOverload(op='aten.add', overload='int_float')>, <OpOverload(op='aten.add', overload='float_int')>, <OpOverload(op='aten.add', overload='default')>]
prims.add <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.add')>, <OpOverload(op='prims.add', overload='default')>]
aten.exp <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.exp')>, <OpOverload(op='aten.exp', overload='default')>, <OpOverload(op='aten.exp', overload='out')>, <OpOverload(op='aten.exp', overload='int')>, <OpOverload(op='aten.exp', overload='float')>, <OpOverload(op='aten.exp', overload='complex')>, <OpOverload(op='aten.exp', overload='Scalar')>]
prims.exp <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.exp')>, <OpOverload(op='prims.exp', overload='default')>]
aten.exp2 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.exp2')>, <OpOverload(op='aten.exp2', overload='default')>, <OpOverload(op='aten.exp2', overload='out')>]
prims.exp2 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.exp2')>, <OpOverload(op='prims.exp2', overload='default')>]
aten.expm1 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.expm1')>, <OpOverload(op='aten.expm1', overload='default')>, <OpOverload(op='aten.expm1', overload='out')>, <OpOverload(op='aten.expm1', overload='int')>, <OpOverload(op='aten.expm1', overload='float')>, <OpOverload(op='aten.expm1', overload='Scalar')>]
prims.expm1 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.expm1')>, <OpOverload(op='prims.expm1', overload='default')>]
aten.relu <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.relu')>, <OpOverload(op='aten.relu', overload='default')>, <OpOverload(op='aten.relu', overload='out')>]
aten.sigmoid <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sigmoid')>, <OpOverload(op='aten.sigmoid', overload='default')>, <OpOverload(op='aten.sigmoid', overload='out')>]
aten.sqrt <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sqrt')>, <OpOverload(op='aten.sqrt', overload='default')>, <OpOverload(op='aten.sqrt', overload='out')>, <OpOverload(op='aten.sqrt', overload='int')>, <OpOverload(op='aten.sqrt', overload='float')>, <OpOverload(op='aten.sqrt', overload='complex')>, <OpOverload(op='aten.sqrt', overload='Scalar')>]
prims.sqrt <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.sqrt')>, <OpOverload(op='prims.sqrt', overload='default')>]
aten.square <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.square')>, <OpOverload(op='aten.square', overload='default')>, <OpOverload(op='aten.square', overload='out')>]
aten.sub <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sub')>, <OpOverload(op='aten.sub', overload='Tensor')>, <OpOverload(op='aten.sub', overload='Scalar')>, <OpOverload(op='aten.sub', overload='out')>, <OpOverload(op='aten.sub', overload='Scalar_out')>, <OpOverload(op='aten.sub', overload='int')>, <OpOverload(op='aten.sub', overload='complex')>, <OpOverload(op='aten.sub', overload='float')>, <OpOverload(op='aten.sub', overload='int_complex')>, <OpOverload(op='aten.sub', overload='complex_int')>, <OpOverload(op='aten.sub', overload='float_complex')>, <OpOverload(op='aten.sub', overload='complex_float')>, <OpOverload(op='aten.sub', overload='int_float')>, <OpOverload(op='aten.sub', overload='float_int')>, <OpOverload(op='aten.sub', overload='default')>]
prims.sub <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.sub')>, <OpOverload(op='prims.sub', overload='default')>]
aten.cos <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.cos')>, <OpOverload(op='aten.cos', overload='default')>, <OpOverload(op='aten.cos', overload='out')>, <OpOverload(op='aten.cos', overload='int')>, <OpOverload(op='aten.cos', overload='float')>, <OpOverload(op='aten.cos', overload='complex')>, <OpOverload(op='aten.cos', overload='Scalar')>]
prims.cos <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.cos')>, <OpOverload(op='prims.cos', overload='default')>]
aten.sin <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sin')>, <OpOverload(op='aten.sin', overload='default')>, <OpOverload(op='aten.sin', overload='out')>, <OpOverload(op='aten.sin', overload='int')>, <OpOverload(op='aten.sin', overload='float')>, <OpOverload(op='aten.sin', overload='complex')>, <OpOverload(op='aten.sin', overload='Scalar')>]
prims.sin <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.sin')>, <OpOverload(op='prims.sin', overload='default')>]
aten.abs <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.abs')>, <OpOverload(op='aten.abs', overload='default')>, <OpOverload(op='aten.abs', overload='out')>]
prims.abs <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.abs')>, <OpOverload(op='prims.abs', overload='default')>]
aten.bitwise_and <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_and')>, <OpOverload(op='aten.bitwise_and', overload='Tensor')>, <OpOverload(op='aten.bitwise_and', overload='Scalar')>, <OpOverload(op='aten.bitwise_and', overload='Scalar_Tensor')>, <OpOverload(op='aten.bitwise_and', overload='Tensor_out')>, <OpOverload(op='aten.bitwise_and', overload='Scalar_out')>, <OpOverload(op='aten.bitwise_and', overload='Scalar_Tensor_out')>]
prims.bitwise_and <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.bitwise_and')>, <OpOverload(op='prims.bitwise_and', overload='default')>]
aten.bitwise_left_shift <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_left_shift')>, <OpOverload(op='aten.bitwise_left_shift', overload='Tensor')>, <OpOverload(op='aten.bitwise_left_shift', overload='Tensor_Scalar')>, <OpOverload(op='aten.bitwise_left_shift', overload='Scalar_Tensor')>, <OpOverload(op='aten.bitwise_left_shift', overload='Tensor_out')>, <OpOverload(op='aten.bitwise_left_shift', overload='Tensor_Scalar_out')>, <OpOverload(op='aten.bitwise_left_shift', overload='Scalar_Tensor_out')>]
aten.bitwise_not <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_not')>, <OpOverload(op='aten.bitwise_not', overload='default')>, <OpOverload(op='aten.bitwise_not', overload='out')>]
prims.bitwise_not <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.bitwise_not')>, <OpOverload(op='prims.bitwise_not', overload='default')>]
aten.bitwise_or <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_or')>, <OpOverload(op='aten.bitwise_or', overload='Tensor')>, <OpOverload(op='aten.bitwise_or', overload='Scalar')>, <OpOverload(op='aten.bitwise_or', overload='Scalar_Tensor')>, <OpOverload(op='aten.bitwise_or', overload='Tensor_out')>, <OpOverload(op='aten.bitwise_or', overload='Scalar_out')>, <OpOverload(op='aten.bitwise_or', overload='Scalar_Tensor_out')>]
prims.bitwise_or <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.bitwise_or')>, <OpOverload(op='prims.bitwise_or', overload='default')>]
aten.bitwise_right_shift <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_right_shift')>, <OpOverload(op='aten.bitwise_right_shift', overload='Tensor')>, <OpOverload(op='aten.bitwise_right_shift', overload='Tensor_Scalar')>, <OpOverload(op='aten.bitwise_right_shift', overload='Scalar_Tensor')>, <OpOverload(op='aten.bitwise_right_shift', overload='Tensor_out')>, <OpOverload(op='aten.bitwise_right_shift', overload='Tensor_Scalar_out')>, <OpOverload(op='aten.bitwise_right_shift', overload='Scalar_Tensor_out')>]
aten.bitwise_xor <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_xor')>, <OpOverload(op='aten.bitwise_xor', overload='Tensor')>, <OpOverload(op='aten.bitwise_xor', overload='Scalar')>, <OpOverload(op='aten.bitwise_xor', overload='Scalar_Tensor')>, <OpOverload(op='aten.bitwise_xor', overload='Tensor_out')>, <OpOverload(op='aten.bitwise_xor', overload='Scalar_out')>, <OpOverload(op='aten.bitwise_xor', overload='Scalar_Tensor_out')>]
prims.bitwise_xor <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.bitwise_xor')>, <OpOverload(op='prims.bitwise_xor', overload='default')>]
aten.lgamma <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.lgamma')>, <OpOverload(op='aten.lgamma', overload='default')>, <OpOverload(op='aten.lgamma', overload='out')>, <OpOverload(op='aten.lgamma', overload='int')>, <OpOverload(op='aten.lgamma', overload='float')>, <OpOverload(op='aten.lgamma', overload='Scalar')>]
prims.lgamma <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.lgamma')>, <OpOverload(op='prims.lgamma', overload='default')>]
aten.erf <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.erf')>, <OpOverload(op='aten.erf', overload='default')>, <OpOverload(op='aten.erf', overload='out')>, <OpOverload(op='aten.erf', overload='int')>, <OpOverload(op='aten.erf', overload='float')>, <OpOverload(op='aten.erf', overload='Scalar')>]
prims.erf <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.erf')>, <OpOverload(op='prims.erf', overload='default')>]
aten.special_erf <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.special_erf')>, <OpOverload(op='aten.special_erf', overload='default')>, <OpOverload(op='aten.special_erf', overload='out')>]
aten.log1p <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.log1p')>, <OpOverload(op='aten.log1p', overload='default')>, <OpOverload(op='aten.log1p', overload='out')>, <OpOverload(op='aten.log1p', overload='int')>, <OpOverload(op='aten.log1p', overload='float')>, <OpOverload(op='aten.log1p', overload='Scalar')>]
prims.log1p <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.log1p')>, <OpOverload(op='prims.log1p', overload='default')>]
aten.tan <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.tan')>, <OpOverload(op='aten.tan', overload='default')>, <OpOverload(op='aten.tan', overload='out')>, <OpOverload(op='aten.tan', overload='int')>, <OpOverload(op='aten.tan', overload='float')>, <OpOverload(op='aten.tan', overload='complex')>, <OpOverload(op='aten.tan', overload='Scalar')>]
prims.tan <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.tan')>, <OpOverload(op='prims.tan', overload='default')>]
aten.tanh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.tanh')>, <OpOverload(op='aten.tanh', overload='default')>, <OpOverload(op='aten.tanh', overload='out')>, <OpOverload(op='aten.tanh', overload='int')>, <OpOverload(op='aten.tanh', overload='float')>, <OpOverload(op='aten.tanh', overload='complex')>, <OpOverload(op='aten.tanh', overload='Scalar')>]
prims.tanh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.tanh')>, <OpOverload(op='prims.tanh', overload='default')>]
aten.log <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.log')>, <OpOverload(op='aten.log', overload='default')>, <OpOverload(op='aten.log', overload='out')>, <OpOverload(op='aten.log', overload='int')>, <OpOverload(op='aten.log', overload='float')>, <OpOverload(op='aten.log', overload='complex')>, <OpOverload(op='aten.log', overload='Scalar')>, <OpOverload(op='aten.log', overload='int_int')>, <OpOverload(op='aten.log', overload='float_float')>, <OpOverload(op='aten.log', overload='complex_complex')>, <OpOverload(op='aten.log', overload='int_float')>, <OpOverload(op='aten.log', overload='float_int')>, <OpOverload(op='aten.log', overload='int_complex')>, <OpOverload(op='aten.log', overload='complex_int')>, <OpOverload(op='aten.log', overload='float_complex')>, <OpOverload(op='aten.log', overload='complex_float')>, <OpOverload(op='aten.log', overload='Scalar_Scalar')>]
prims.log <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.log')>, <OpOverload(op='prims.log', overload='default')>]
aten.logical_and <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.logical_and')>, <OpOverload(op='aten.logical_and', overload='default')>, <OpOverload(op='aten.logical_and', overload='out')>]
aten.logical_not <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.logical_not')>, <OpOverload(op='aten.logical_not', overload='default')>, <OpOverload(op='aten.logical_not', overload='out')>]
aten.logical_or <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.logical_or')>, <OpOverload(op='aten.logical_or', overload='default')>, <OpOverload(op='aten.logical_or', overload='out')>]
aten.logical_xor <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.logical_xor')>, <OpOverload(op='aten.logical_xor', overload='default')>, <OpOverload(op='aten.logical_xor', overload='out')>]
aten.maximum <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.maximum')>, <OpOverload(op='aten.maximum', overload='default')>, <OpOverload(op='aten.maximum', overload='out')>]
prims.maximum <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.maximum')>, <OpOverload(op='prims.maximum', overload='default')>]
aten.minimum <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.minimum')>, <OpOverload(op='aten.minimum', overload='default')>, <OpOverload(op='aten.minimum', overload='out')>]
prims.minimum <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.minimum')>, <OpOverload(op='prims.minimum', overload='default')>]
aten.clamp_min <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.clamp_min')>, <OpOverload(op='aten.clamp_min', overload='default')>, <OpOverload(op='aten.clamp_min', overload='Tensor')>, <OpOverload(op='aten.clamp_min', overload='out')>, <OpOverload(op='aten.clamp_min', overload='Tensor_out')>]
aten.clamp_max <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.clamp_max')>, <OpOverload(op='aten.clamp_max', overload='default')>, <OpOverload(op='aten.clamp_max', overload='Tensor')>, <OpOverload(op='aten.clamp_max', overload='out')>, <OpOverload(op='aten.clamp_max', overload='Tensor_out')>]
aten.neg <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.neg')>, <OpOverload(op='aten.neg', overload='default')>, <OpOverload(op='aten.neg', overload='out')>, <OpOverload(op='aten.neg', overload='int')>, <OpOverload(op='aten.neg', overload='float')>, <OpOverload(op='aten.neg', overload='complex')>, <OpOverload(op='aten.neg', overload='Scalar')>]
prims.neg <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.neg')>, <OpOverload(op='prims.neg', overload='default')>]
aten.abs <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.abs')>]
prims.abs <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.abs')>]
aten.reciprocal <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.reciprocal')>, <OpOverload(op='aten.reciprocal', overload='default')>, <OpOverload(op='aten.reciprocal', overload='out')>]
prims.reciprocal <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.reciprocal')>, <OpOverload(op='prims.reciprocal', overload='default')>]
aten.remainder <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.remainder')>, <OpOverload(op='aten.remainder', overload='Tensor')>, <OpOverload(op='aten.remainder', overload='Scalar')>, <OpOverload(op='aten.remainder', overload='Scalar_Tensor')>, <OpOverload(op='aten.remainder', overload='Tensor_out')>, <OpOverload(op='aten.remainder', overload='Scalar_out')>, <OpOverload(op='aten.remainder', overload='Scalar_Tensor_out')>, <OpOverload(op='aten.remainder', overload='int')>, <OpOverload(op='aten.remainder', overload='float')>, <OpOverload(op='aten.remainder', overload='int_float')>, <OpOverload(op='aten.remainder', overload='float_int')>, <OpOverload(op='aten.remainder', overload='default')>]
prims.remainder <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.remainder')>, <OpOverload(op='prims.remainder', overload='default')>]
aten.sign <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sign')>, <OpOverload(op='aten.sign', overload='default')>, <OpOverload(op='aten.sign', overload='out')>]
prims.sign <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.sign')>, <OpOverload(op='prims.sign', overload='default')>]
aten.ceil <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.ceil')>]
prims.ceil <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.ceil')>, <OpOverload(op='prims.ceil', overload='default')>]
aten.signbit <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.signbit')>, <OpOverload(op='aten.signbit', overload='default')>, <OpOverload(op='aten.signbit', overload='out')>]
prims.signbit <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.signbit')>, <OpOverload(op='prims.signbit', overload='default')>]
aten._neg_view <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._neg_view')>, <OpOverload(op='aten._neg_view', overload='default')>]
aten.le <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.le')>, <OpOverload(op='aten.le', overload='Tensor')>, <OpOverload(op='aten.le', overload='Scalar')>, <OpOverload(op='aten.le', overload='Scalar_out')>, <OpOverload(op='aten.le', overload='Tensor_out')>, <OpOverload(op='aten.le', overload='int')>, <OpOverload(op='aten.le', overload='float')>, <OpOverload(op='aten.le', overload='int_float')>, <OpOverload(op='aten.le', overload='float_int')>, <OpOverload(op='aten.le', overload='default')>, <OpOverload(op='aten.le', overload='str')>]
prims.le <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.le')>, <OpOverload(op='prims.le', overload='default')>]
aten.lt <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.lt')>, <OpOverload(op='aten.lt', overload='Tensor')>, <OpOverload(op='aten.lt', overload='Scalar')>, <OpOverload(op='aten.lt', overload='Scalar_out')>, <OpOverload(op='aten.lt', overload='Tensor_out')>, <OpOverload(op='aten.lt', overload='int')>, <OpOverload(op='aten.lt', overload='float')>, <OpOverload(op='aten.lt', overload='int_float')>, <OpOverload(op='aten.lt', overload='float_int')>, <OpOverload(op='aten.lt', overload='default')>, <OpOverload(op='aten.lt', overload='str')>]
prims.lt <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.lt')>, <OpOverload(op='prims.lt', overload='default')>]
aten.ge <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.ge')>, <OpOverload(op='aten.ge', overload='Tensor')>, <OpOverload(op='aten.ge', overload='Scalar')>, <OpOverload(op='aten.ge', overload='Scalar_out')>, <OpOverload(op='aten.ge', overload='Tensor_out')>, <OpOverload(op='aten.ge', overload='int')>, <OpOverload(op='aten.ge', overload='float')>, <OpOverload(op='aten.ge', overload='int_float')>, <OpOverload(op='aten.ge', overload='float_int')>, <OpOverload(op='aten.ge', overload='default')>, <OpOverload(op='aten.ge', overload='str')>]
prims.ge <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.ge')>, <OpOverload(op='prims.ge', overload='default')>]
aten.gt <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.gt')>, <OpOverload(op='aten.gt', overload='Tensor')>, <OpOverload(op='aten.gt', overload='Scalar')>, <OpOverload(op='aten.gt', overload='Scalar_out')>, <OpOverload(op='aten.gt', overload='Tensor_out')>, <OpOverload(op='aten.gt', overload='int')>, <OpOverload(op='aten.gt', overload='float')>, <OpOverload(op='aten.gt', overload='int_float')>, <OpOverload(op='aten.gt', overload='float_int')>, <OpOverload(op='aten.gt', overload='default')>, <OpOverload(op='aten.gt', overload='str')>]
prims.gt <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.gt')>, <OpOverload(op='prims.gt', overload='default')>]
aten.eq <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.eq')>, <OpOverload(op='aten.eq', overload='Tensor')>, <OpOverload(op='aten.eq', overload='Scalar')>, <OpOverload(op='aten.eq', overload='Scalar_out')>, <OpOverload(op='aten.eq', overload='Tensor_out')>, <OpOverload(op='aten.eq', overload='int_list')>, <OpOverload(op='aten.eq', overload='device')>, <OpOverload(op='aten.eq', overload='bool')>, <OpOverload(op='aten.eq', overload='enum')>, <OpOverload(op='aten.eq', overload='int')>, <OpOverload(op='aten.eq', overload='complex')>, <OpOverload(op='aten.eq', overload='float')>, <OpOverload(op='aten.eq', overload='int_float')>, <OpOverload(op='aten.eq', overload='float_int')>, <OpOverload(op='aten.eq', overload='float_complex')>, <OpOverload(op='aten.eq', overload='complex_float')>, <OpOverload(op='aten.eq', overload='default')>, <OpOverload(op='aten.eq', overload='str')>, <OpOverload(op='aten.eq', overload='float_list')>, <OpOverload(op='aten.eq', overload='Tensor_list')>, <OpOverload(op='aten.eq', overload='bool_list')>, <OpOverload(op='aten.eq', overload='str_list')>]
prims.eq <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.eq')>, <OpOverload(op='prims.eq', overload='default')>]
aten.ne <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.ne')>, <OpOverload(op='aten.ne', overload='Tensor')>, <OpOverload(op='aten.ne', overload='Scalar')>, <OpOverload(op='aten.ne', overload='Scalar_out')>, <OpOverload(op='aten.ne', overload='Tensor_out')>, <OpOverload(op='aten.ne', overload='int_list')>, <OpOverload(op='aten.ne', overload='device')>, <OpOverload(op='aten.ne', overload='bool')>, <OpOverload(op='aten.ne', overload='enum')>, <OpOverload(op='aten.ne', overload='int')>, <OpOverload(op='aten.ne', overload='complex')>, <OpOverload(op='aten.ne', overload='float')>, <OpOverload(op='aten.ne', overload='int_float')>, <OpOverload(op='aten.ne', overload='float_int')>, <OpOverload(op='aten.ne', overload='float_complex')>, <OpOverload(op='aten.ne', overload='complex_float')>, <OpOverload(op='aten.ne', overload='default')>, <OpOverload(op='aten.ne', overload='str')>, <OpOverload(op='aten.ne', overload='float_list')>, <OpOverload(op='aten.ne', overload='Tensor_list')>, <OpOverload(op='aten.ne', overload='bool_list')>, <OpOverload(op='aten.ne', overload='str_list')>]
prims.ne <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.ne')>, <OpOverload(op='prims.ne', overload='default')>]
aten.cosh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.cosh')>, <OpOverload(op='aten.cosh', overload='default')>, <OpOverload(op='aten.cosh', overload='out')>, <OpOverload(op='aten.cosh', overload='int')>, <OpOverload(op='aten.cosh', overload='float')>, <OpOverload(op='aten.cosh', overload='complex')>, <OpOverload(op='aten.cosh', overload='Scalar')>]
prims.cosh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.cosh')>, <OpOverload(op='prims.cosh', overload='default')>]
aten.sinh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sinh')>, <OpOverload(op='aten.sinh', overload='default')>, <OpOverload(op='aten.sinh', overload='out')>, <OpOverload(op='aten.sinh', overload='int')>, <OpOverload(op='aten.sinh', overload='float')>, <OpOverload(op='aten.sinh', overload='complex')>, <OpOverload(op='aten.sinh', overload='Scalar')>]
prims.sinh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.sinh')>, <OpOverload(op='prims.sinh', overload='default')>]
aten.acos <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.acos')>, <OpOverload(op='aten.acos', overload='default')>, <OpOverload(op='aten.acos', overload='out')>, <OpOverload(op='aten.acos', overload='int')>, <OpOverload(op='aten.acos', overload='float')>, <OpOverload(op='aten.acos', overload='complex')>, <OpOverload(op='aten.acos', overload='Scalar')>]
prims.acos <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.acos')>, <OpOverload(op='prims.acos', overload='default')>]
aten.acosh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.acosh')>, <OpOverload(op='aten.acosh', overload='default')>, <OpOverload(op='aten.acosh', overload='out')>, <OpOverload(op='aten.acosh', overload='int')>, <OpOverload(op='aten.acosh', overload='float')>, <OpOverload(op='aten.acosh', overload='complex')>, <OpOverload(op='aten.acosh', overload='Scalar')>]
prims.acosh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.acosh')>, <OpOverload(op='prims.acosh', overload='default')>]
aten.asin <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.asin')>, <OpOverload(op='aten.asin', overload='default')>, <OpOverload(op='aten.asin', overload='out')>, <OpOverload(op='aten.asin', overload='int')>, <OpOverload(op='aten.asin', overload='float')>, <OpOverload(op='aten.asin', overload='complex')>, <OpOverload(op='aten.asin', overload='Scalar')>]
prims.asin <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.asin')>, <OpOverload(op='prims.asin', overload='default')>]
aten.asinh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.asinh')>, <OpOverload(op='aten.asinh', overload='default')>, <OpOverload(op='aten.asinh', overload='out')>, <OpOverload(op='aten.asinh', overload='int')>, <OpOverload(op='aten.asinh', overload='float')>, <OpOverload(op='aten.asinh', overload='complex')>, <OpOverload(op='aten.asinh', overload='Scalar')>]
prims.asinh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.asinh')>, <OpOverload(op='prims.asinh', overload='default')>]
aten.atan2 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.atan2')>, <OpOverload(op='aten.atan2', overload='default')>, <OpOverload(op='aten.atan2', overload='out')>, <OpOverload(op='aten.atan2', overload='int')>, <OpOverload(op='aten.atan2', overload='float')>, <OpOverload(op='aten.atan2', overload='int_float')>, <OpOverload(op='aten.atan2', overload='float_int')>, <OpOverload(op='aten.atan2', overload='Scalar_Scalar')>]
prims.atan2 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.atan2')>, <OpOverload(op='prims.atan2', overload='default')>]
aten.atan <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.atan')>, <OpOverload(op='aten.atan', overload='default')>, <OpOverload(op='aten.atan', overload='out')>, <OpOverload(op='aten.atan', overload='int')>, <OpOverload(op='aten.atan', overload='float')>, <OpOverload(op='aten.atan', overload='complex')>, <OpOverload(op='aten.atan', overload='Scalar')>]
prims.atan <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.atan')>, <OpOverload(op='prims.atan', overload='default')>]
aten.atanh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.atanh')>, <OpOverload(op='aten.atanh', overload='default')>, <OpOverload(op='aten.atanh', overload='out')>, <OpOverload(op='aten.atanh', overload='int')>, <OpOverload(op='aten.atanh', overload='float')>, <OpOverload(op='aten.atanh', overload='complex')>, <OpOverload(op='aten.atanh', overload='Scalar')>]
prims.atanh <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.atanh')>, <OpOverload(op='prims.atanh', overload='default')>]
aten.copysign <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.copysign')>, <OpOverload(op='aten.copysign', overload='Tensor')>, <OpOverload(op='aten.copysign', overload='Scalar')>, <OpOverload(op='aten.copysign', overload='out')>, <OpOverload(op='aten.copysign', overload='Scalar_out')>, <OpOverload(op='aten.copysign', overload='int')>, <OpOverload(op='aten.copysign', overload='float')>, <OpOverload(op='aten.copysign', overload='int_float')>, <OpOverload(op='aten.copysign', overload='float_int')>, <OpOverload(op='aten.copysign', overload='default')>]
aten.erfc <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.erfc')>, <OpOverload(op='aten.erfc', overload='default')>, <OpOverload(op='aten.erfc', overload='out')>, <OpOverload(op='aten.erfc', overload='int')>, <OpOverload(op='aten.erfc', overload='float')>, <OpOverload(op='aten.erfc', overload='Scalar')>]
prims.erfc <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.erfc')>, <OpOverload(op='prims.erfc', overload='default')>]
aten.erfinv <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.erfinv')>, <OpOverload(op='aten.erfinv', overload='default')>, <OpOverload(op='aten.erfinv', overload='out')>]
aten.hypot <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.hypot')>, <OpOverload(op='aten.hypot', overload='default')>, <OpOverload(op='aten.hypot', overload='out')>]
prims.hypot <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.hypot')>, <OpOverload(op='prims.hypot', overload='default')>]
aten.log10 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.log10')>, <OpOverload(op='aten.log10', overload='default')>, <OpOverload(op='aten.log10', overload='out')>, <OpOverload(op='aten.log10', overload='int')>, <OpOverload(op='aten.log10', overload='float')>, <OpOverload(op='aten.log10', overload='complex')>, <OpOverload(op='aten.log10', overload='Scalar')>]
prims.log10 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.log10')>, <OpOverload(op='prims.log10', overload='default')>]
aten.nextafter <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.nextafter')>, <OpOverload(op='aten.nextafter', overload='default')>, <OpOverload(op='aten.nextafter', overload='out')>]
prims.nextafter <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.nextafter')>, <OpOverload(op='prims.nextafter', overload='default')>]
prims.bessel_j0 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.bessel_j0')>, <OpOverload(op='prims.bessel_j0', overload='default')>]
prims.bessel_j0 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.bessel_j0')>]
prims.bessel_j1 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.bessel_j1')>, <OpOverload(op='prims.bessel_j1', overload='default')>]
prims.bessel_j1 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.bessel_j1')>]
prims.digamma <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.digamma')>, <OpOverload(op='prims.digamma', overload='default')>]
prims.digamma <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.digamma')>]
prims.digamma <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.digamma')>]
prims.erfcx <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.erfcx')>, <OpOverload(op='prims.erfcx', overload='default')>]
prims.erfcx <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.erfcx')>]
prims.igamma <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.igamma')>, <OpOverload(op='prims.igamma', overload='default')>]
prims.igamma <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.igamma')>]
prims.igamma <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.igamma')>]
prims.igammac <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.igammac')>, <OpOverload(op='prims.igammac', overload='default')>]
prims.igammac <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.igammac')>]
prims.igammac <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.igammac')>]
prims.ndtri <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.ndtri')>, <OpOverload(op='prims.ndtri', overload='default')>]
prims.ndtri <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.ndtri')>]
prims.spherical_bessel_j0 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.spherical_bessel_j0')>, <OpOverload(op='prims.spherical_bessel_j0', overload='default')>]
prims.spherical_bessel_j0 <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.spherical_bessel_j0')>]
prims.zeta <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.zeta')>, <OpOverload(op='prims.zeta', overload='default')>]
prims.zeta <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.zeta')>]
prims.zeta <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.zeta')>]
prims.zeta <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.zeta')>]
prims.zeta <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.zeta')>]
prims.zeta <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='prims.zeta')>]
aten.add_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.add_')>, <OpOverload(op='aten.add_', overload='Tensor')>, <OpOverload(op='aten.add_', overload='Scalar')>, <OpOverload(op='aten.add_', overload='t')>]
aten.bitwise_and_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_and_')>, <OpOverload(op='aten.bitwise_and_', overload='Tensor')>, <OpOverload(op='aten.bitwise_and_', overload='Scalar')>]
aten.bitwise_left_shift_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_left_shift_')>, <OpOverload(op='aten.bitwise_left_shift_', overload='Tensor_Scalar')>, <OpOverload(op='aten.bitwise_left_shift_', overload='Tensor')>]
aten.bitwise_not_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_not_')>, <OpOverload(op='aten.bitwise_not_', overload='default')>]
aten.bitwise_or_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_or_')>, <OpOverload(op='aten.bitwise_or_', overload='Tensor')>, <OpOverload(op='aten.bitwise_or_', overload='Scalar')>]
aten.bitwise_right_shift_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_right_shift_')>, <OpOverload(op='aten.bitwise_right_shift_', overload='Tensor_Scalar')>, <OpOverload(op='aten.bitwise_right_shift_', overload='Tensor')>]
aten.bitwise_xor_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bitwise_xor_')>, <OpOverload(op='aten.bitwise_xor_', overload='Tensor')>, <OpOverload(op='aten.bitwise_xor_', overload='Scalar')>]
aten.mul_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.mul_')>, <OpOverload(op='aten.mul_', overload='Tensor')>, <OpOverload(op='aten.mul_', overload='Scalar')>, <OpOverload(op='aten.mul_', overload='t')>]
aten.logical_and_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.logical_and_')>, <OpOverload(op='aten.logical_and_', overload='default')>]
aten.logical_not_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.logical_not_')>, <OpOverload(op='aten.logical_not_', overload='default')>]
aten.logical_or_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.logical_or_')>, <OpOverload(op='aten.logical_or_', overload='default')>]
aten.logical_xor_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.logical_xor_')>, <OpOverload(op='aten.logical_xor_', overload='default')>]
aten.sub_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sub_')>, <OpOverload(op='aten.sub_', overload='Tensor')>, <OpOverload(op='aten.sub_', overload='Scalar')>]
aten.relu_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.relu_')>, <OpOverload(op='aten.relu_', overload='default')>]
aten.sigmoid_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sigmoid_')>, <OpOverload(op='aten.sigmoid_', overload='default')>]
aten.__and__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__and__')>, <OpOverload(op='aten.__and__', overload='Scalar')>, <OpOverload(op='aten.__and__', overload='Tensor')>, <OpOverload(op='aten.__and__', overload='bool')>, <OpOverload(op='aten.__and__', overload='int')>]
aten.__lshift__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__lshift__')>, <OpOverload(op='aten.__lshift__', overload='Tensor')>, <OpOverload(op='aten.__lshift__', overload='Scalar')>, <OpOverload(op='aten.__lshift__', overload='Scalar_out')>, <OpOverload(op='aten.__lshift__', overload='Tensor_out')>, <OpOverload(op='aten.__lshift__', overload='int')>]
aten.__or__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__or__')>, <OpOverload(op='aten.__or__', overload='Tensor')>, <OpOverload(op='aten.__or__', overload='Scalar')>, <OpOverload(op='aten.__or__', overload='bool')>, <OpOverload(op='aten.__or__', overload='int')>]
aten.__rshift__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__rshift__')>, <OpOverload(op='aten.__rshift__', overload='Tensor')>, <OpOverload(op='aten.__rshift__', overload='Scalar')>, <OpOverload(op='aten.__rshift__', overload='Scalar_out')>, <OpOverload(op='aten.__rshift__', overload='Tensor_out')>, <OpOverload(op='aten.__rshift__', overload='int')>]
aten.__xor__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__xor__')>, <OpOverload(op='aten.__xor__', overload='Tensor')>, <OpOverload(op='aten.__xor__', overload='Scalar')>, <OpOverload(op='aten.__xor__', overload='bool')>, <OpOverload(op='aten.__xor__', overload='int')>]
aten.__iand__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__iand__')>, <OpOverload(op='aten.__iand__', overload='Tensor')>, <OpOverload(op='aten.__iand__', overload='Scalar')>]
aten.__ilshift__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__ilshift__')>, <OpOverload(op='aten.__ilshift__', overload='Tensor')>, <OpOverload(op='aten.__ilshift__', overload='Scalar')>]
aten.__ior__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__ior__')>, <OpOverload(op='aten.__ior__', overload='Tensor')>, <OpOverload(op='aten.__ior__', overload='Scalar')>]
aten.__irshift__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__irshift__')>, <OpOverload(op='aten.__irshift__', overload='Tensor')>, <OpOverload(op='aten.__irshift__', overload='Scalar')>]
aten.__ixor__ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.__ixor__')>, <OpOverload(op='aten.__ixor__', overload='Tensor')>, <OpOverload(op='aten.__ixor__', overload='Scalar')>]
aten.sym_constrain_range <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sym_constrain_range')>, <OpOverload(op='aten.sym_constrain_range', overload='default')>]
aten.sym_numel <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.sym_numel')>, <OpOverload(op='aten.sym_numel', overload='default')>]
<built-in function add> <class 'builtin_function_or_method'> [<built-in function add>]
<built-in function sub> <class 'builtin_function_or_method'> [<built-in function sub>]
<built-in function mul> <class 'builtin_function_or_method'> [<built-in function mul>]
<built-in function mod> <class 'builtin_function_or_method'> [<built-in function mod>]
<built-in function pow> <class 'builtin_function_or_method'> [<built-in function pow>]
<built-in function and_> <class 'builtin_function_or_method'> [<built-in function and_>]
<built-in function or_> <class 'builtin_function_or_method'> [<built-in function or_>]
<built-in function truediv> <class 'builtin_function_or_method'> [<built-in function truediv>]
<built-in function floordiv> <class 'builtin_function_or_method'> [<built-in function floordiv>]
<built-in function lshift> <class 'builtin_function_or_method'> [<built-in function lshift>]
<built-in function rshift> <class 'builtin_function_or_method'> [<built-in function rshift>]
<function sym_not at 0x7ff281d7d3a0> <class 'function'> [<function sym_not at 0x7ff281d7d3a0>]
<built-in function pos> <class 'builtin_function_or_method'> [<built-in function pos>]
<built-in function eq> <class 'builtin_function_or_method'> [<built-in function eq>]
<built-in function ne> <class 'builtin_function_or_method'> [<built-in function ne>]
<built-in function gt> <class 'builtin_function_or_method'> [<built-in function gt>]
<built-in function lt> <class 'builtin_function_or_method'> [<built-in function lt>]
<built-in function le> <class 'builtin_function_or_method'> [<built-in function le>]
<built-in function ge> <class 'builtin_function_or_method'> [<built-in function ge>]
<built-in function floor> <class 'builtin_function_or_method'> [<built-in function floor>]
<function sym_float at 0x7ff201312790> <class 'function'> [<function sym_float at 0x7ff201312790>]
<built-in function ceil> <class 'builtin_function_or_method'> [<built-in function ceil>]
<built-in function neg> <class 'builtin_function_or_method'> [<built-in function neg>]
<function sym_min at 0x7ff201312940> <class 'function'> [<function sym_min at 0x7ff201312940>]
<function sym_max at 0x7ff2013128b0> <class 'function'> [<function sym_max at 0x7ff2013128b0>]
<function sym_ite at 0x7ff201315040> <class 'function'> [<function sym_ite at 0x7ff201315040>]
<built-in function abs> <class 'builtin_function_or_method'> [<built-in function abs>]
<built-in function round> <class 'builtin_function_or_method'> [<built-in function round>]
<function <lambda> at 0x7ff1ff8923a0> <class 'function'> [<function <lambda> at 0x7ff1ff8923a0>]
<function _sym_sqrt at 0x7ff201312a60> <class 'function'> [<function _sym_sqrt at 0x7ff201312a60>]
<function _sym_cos at 0x7ff201312af0> <class 'function'> [<function _sym_cos at 0x7ff201312af0>]
<function _sym_cosh at 0x7ff201312b80> <class 'function'> [<function _sym_cosh at 0x7ff201312b80>]
<function _sym_sin at 0x7ff201312c10> <class 'function'> [<function _sym_sin at 0x7ff201312c10>]
<function _sym_sinh at 0x7ff201312ca0> <class 'function'> [<function _sym_sinh at 0x7ff201312ca0>]
<function _sym_tan at 0x7ff201312d30> <class 'function'> [<function _sym_tan at 0x7ff201312d30>]
<function _sym_tanh at 0x7ff201312dc0> <class 'function'> [<function _sym_tanh at 0x7ff201312dc0>]
<function _sym_asin at 0x7ff201312e50> <class 'function'> [<function _sym_asin at 0x7ff201312e50>]
<function _sym_acos at 0x7ff201312ee0> <class 'function'> [<function _sym_acos at 0x7ff201312ee0>]
<function _sym_atan at 0x7ff201312f70> <class 'function'> [<function _sym_atan at 0x7ff201312f70>]
aten._foobar <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._foobar')>]
_inductor_test.realize <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='_inductor_test.realize')>, <OpOverload(op='_inductor_test.realize', overload='default')>]
inductor.resize_storage_bytes_ <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='inductor.resize_storage_bytes_')>, <OpOverload(op='inductor.resize_storage_bytes_', overload='default')>]
auto_functionalized <class 'torch._higher_order_ops.auto_functionalize.AutoFunctionalized'> [<torch._higher_order_ops.auto_functionalize.AutoFunctionalized object at 0x7ff1f3cd9d00>]
triton_kernel_wrapper_mutation <class 'torch._higher_order_ops.triton_kernel_wrap.TritonKernelWrapperMutation'> [<torch._higher_order_ops.triton_kernel_wrap.TritonKernelWrapperMutation object at 0x7ff1f3e204c0>]
triton_kernel_wrapper_functional <class 'torch._higher_order_ops.triton_kernel_wrap.TritonKernelWrapperFunctional'> [<torch._higher_order_ops.triton_kernel_wrap.TritonKernelWrapperFunctional object at 0x7ff1f3e205e0>]
cond <class 'torch._ops.HigherOrderOperator'> [<torch._ops.HigherOrderOperator object at 0x7ff1ff7f0d00>]
aten.mm <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.mm')>, <OpOverload(op='aten.mm', overload='default')>, <OpOverload(op='aten.mm', overload='out')>]
aten._int_mm <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._int_mm')>, <OpOverload(op='aten._int_mm', overload='default')>, <OpOverload(op='aten._int_mm', overload='out')>]
aten.addmm <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.addmm')>, <OpOverload(op='aten.addmm', overload='default')>, <OpOverload(op='aten.addmm', overload='out')>]
aten.bmm <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.bmm')>, <OpOverload(op='aten.bmm', overload='default')>, <OpOverload(op='aten.bmm', overload='out')>]
aten.convolution <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten.convolution')>, <OpOverload(op='aten.convolution', overload='default')>, <OpOverload(op='aten.convolution', overload='out')>]
aten._convolution <class 'torch._ops.OpOverloadPacket'> [<OpOverloadPacket(op='aten._convolution')>, <OpOverload(op='aten._convolution', overload='deprecated')>, <OpOverload(op='aten._convolution', overload='default')>, <OpOverload(op='aten._convolution', overload='out')>]
```

</details>

Reply:
> We also allow for registering ops for OpOverloadPackets and these are very commonly used, so we should also check those at least.

OpOverloadPacket does not have py_kernels, if I understand correctly